backend/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM pennlabs/django-base:9c4f31bf1af44219d0f9019271a0033a222291c2-3.10.1
+FROM pennlabs/django-base:b269ea1613686b1ac6370154debbb741b012de1a-3.11
```

Comment:
This hash would not work because it tries to install poetry. Let me give you another one